.goreleaser.yml

```diff
@@ -59,15 +63,83 @@ release:
   # (we want Releaser to be responsible for doing all the GitHub release manipulations)
 
 dockers:
+  # i386
   - image_templates:
-    - "launchdarkly/ld-relay:{{ .Version }}"
-    - "launchdarkly/ld-relay:v{{ .Major }}"
-    - "launchdarkly/ld-relay:latest"
+    - "launchdarkly/ld-relay:{{ .Version }}-i386"
+    - "launchdarkly/ld-relay:v{{ .Major }}-i386"
+    - "launchdarkly/ld-relay:latest-i386"
```

Comment:
If we make a new branch and forget to remove this bit, that's going to cause problems when we release older versions right?

Reply:
(for posterity: discussed offline that it's not optimal that the `latest` tag flip flops. Have another PR up to fix this.)